SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -131,15 +132,18 @@ void USpatialGameInstance::StartSpatialConnection()
 		TryInjectSpatialLocatorIntoCommandLine();
 		CreateNewSpatialConnectionManager();
 	}
-#if TRACE_LIB_ACTIVE
 	else
 	{
 		// In native, setup worker name here as we don't get a HandleOnConnected() callback
-		FString WorkerName =
-			FString::Printf(TEXT("%s:%s"), *SpatialWorkerType.ToString(), *FGuid::NewGuid().ToString(EGuidFormats::Digits));
+		FString WorkerName = FString::Printf(TEXT("%s%s"), *SpatialWorkerType.ToString(), *FGuid::NewGuid().ToString(EGuidFormats::Digits));
+#if TRACE_LIB_ACTIVE
 		SpatialLatencyTracer->SetWorkerId(WorkerName);
-	}
 #endif
+		if (GetDefault<USpatialGDKSettings>()->bEventTracingEnabled)
```

Comment:
Guess it's nice to have the redundancy of the check here and inside the CreateEventTracer?